poms/bom/pom.xml

```diff
@@ -2069,16 +2069,16 @@
                 <artifactId>spring-tx</artifactId>
                 <version>${spring.version}</version>
             </dependency>
-            <dependency>
-                <groupId>org.yaml</groupId>
-                <artifactId>snakeyaml</artifactId>
-                <version>${snakeyaml.version}</version>
-            </dependency>
             <dependency>
                 <groupId>org.threeten</groupId>
                 <artifactId>threetenbp</artifactId>
                 <version>1.4.0</version>
```

Comment:
we should probably replace versions with properties

Reply:
Yes, I'll do that. Thought we had some Groovy hackery which checked that stuff...

Reply:
IIRC, not at the pom level but only at extension/integration test

Reply:
+1 for the prop.